templates/taxi/car_detail.html

```diff
@@ -1,22 +1,15 @@
-{% extends "base.html" %}
-
-{% block content %}
-  <h1>
-    {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
-    </a>
-
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
-    </a>
-  </h1>
-  <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
-  <hr>
-  <ul>
-    {% for driver in car.drivers.all %}
-      <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
-    {% endfor %}
-  </ul>
-{% endblock %}
+{% if request.user.is_driver %}
```

Comment:
The condition `request.user.is_driver` may not correctly verify if the user has the driver role. Ensure that the logic accurately checks the user's role or permissions to determine if they can assign or remove themselves as a driver.